guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -39,8 +39,8 @@ class MyClass implements \Magento\Framework\View\Element\Block\ArgumentInterface
 {
   public function getTitle()
   {
-    return 'Hello World'
-  {
+    return 'Hello World';
```

Comment:
Hi @luanalves. This change has been already proposed in [another](https://github.com/magento/devdocs/pull/5849/files) pull request. Feel free to remove this change from the current pull request.
Thank you